lib/galaxy/jobs/__init__.py

```diff
@@ -1839,13 +1822,7 @@ def finish( self, stdout, stderr, tool_exit_code=None ):
             task.state = task.states.ERROR
 
         # Save stdout and stderr
-        if len( stdout ) > DATABASE_MAX_STRING_SIZE:
-            log.error( "stdout for task %d is greater than %s, only a portion will be logged to database" % ( task.id, DATABASE_MAX_STRING_SIZE_PRETTY ) )
```

Comment:
Why was `log.error()` used here? :open_mouth: 
Anyway, now it's rightfully replaced with `log.info()`.